omnigibson/transition_rules.py

```diff
@@ -347,78 +346,22 @@ def __init__(self, filter_1_name, filter_2_name):
         self._filter_1_name = filter_1_name
         self._filter_2_name = filter_2_name
 
-        # Will be filled in during self.initialize
```

Comment:
Is the final decision to undo this?

Reply:
@ChengshuLi eric can you confirm? Can revert this if you're ok with it

Reply:
@cgokmen reverted back to RigidContactAPI, just confirmed with eric